tests/compiler/t_foreach.test.ts

```diff
@@ -105,6 +105,54 @@ describe("t-foreach", () => {
     expect(renderToString(template, context)).toBe(expected);
   });
 
+  test("iterate, Map param", () => {
+    const template = `
+        <div>
```

Comment:
not really important, but you can remove the `<div>` if you want. not really relevant to the test